substrate/dex/pallet/src/types.rs

```diff
@@ -23,6 +23,9 @@ use super::*;
 use codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
 
+/// 1hr worth of blocks.
+pub const ORACLE_WINDOW_SIZE: u32 = 600;
```

Comment:
We should document how this needs to be long enough for arbitrage to occur and make holding any fake price up sufficiently unprofitable.

Also, this isn't long enough for that AFAICT. It'd take longer than this just to add BTC to Serai to swap it.